packages/tooling/src/lib/parameters-to-json-schema.js

```diff
@@ -46,7 +46,7 @@ function getBodyParam(pathOperation, oas) {
     type,
     label: types[type],
     schema: oas.components
-      ? { definitions: { components: cleanupSchemaDefaults(oas.components) }, ...cleanupSchemaDefaults(schema.schema) }
+      ? { components: cleanupSchemaDefaults(oas.components), ...cleanupSchemaDefaults(schema.schema) }
```

Comment:
Is this how you're supposed to do it in OAS? Did I misread the spec?

Reply:
Nope! This change was needed because the version of your `#/definitions` handling in our fork got merged last week. https://github.com/rjsf-team/react-jsonschema-form/pull/1506

RJSF can now properly access `#/components/schemas/...` without trying to think that they should be located under `#/definitions/components/schemas/...`.